pegasus/sites.v3/code.org/public/private/poste/index.haml

```diff
@@ -83,7 +83,8 @@
     Showing most recently sent mail. Click to view all mail.
 
 :ruby
-  rows = DB.fetch("
+  POSTE_REPORTING_DB_READONLY = sequel_connect(CDO.pegasus_reporting_db_reader, CDO.pegasus_reporting_db_reader)
```

Comment:
Should we define this in one place, instead of each file it's being used? Maybe [here](https://github.com/code-dot-org/code-dot-org/blob/98377d10a53d614e82fa0520e52ab9c9fd8e9f9f/lib/cdo/db.rb#L23-L22) or [here](https://github.com/code-dot-org/code-dot-org/blob/34e8a103c0af2ea3e22adde06883a7d242538ede/lib/cdo/properties.rb#L2-L2)